llvm/docs/ReleaseNotes.rst

```diff
@@ -70,6 +70,8 @@ Changes to the LLVM IR
 
 * Added `llvm.exp10` intrinsic.
 
+* Added a code model attribute for the global variable.
```

Comment:
Helpful to include link to the LangRef reference

Reply:
Thanks